ActionLogger/src/pl/jtat/logger/model/EventData.java

```diff
@@ -0,0 +1,62 @@
+package pl.jtat.logger.model;
+
+import pl.jtat.logger.C;
+import pl.jtat.logger.util.DateFormatter;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+public class EventData {
```

Comment:
To powinno stać się modelem

Reply:
A kiedy wg Ciebie klasa staje się modelem? Mam ją postawić przed fotografem? :) W przypadku tej klasy znajduje się ona w package model w module zawierającym funkcjonalność (po przeniesieniu to i tak będzie osobny package dla loggera z pakietem model).

Reply:
W standardzie Java Persistence API klasy, które mogą być mapowane muszą posiadać adnotacje @Entity na poziomie klasy